src/main/resources/templates/db_scripts/migration/organizations.sql

```diff
@@ -3,3 +3,9 @@ SET
   jsonb = jsonb || jsonb_build_object('isVendor', false)
 WHERE
   NOT jsonb ? 'isVendor';
+
+UPDATE ${myuniversity}_${mymodule}.organizations
+SET
+  jsonb = jsonb || jsonb_build_object('code', 'DEFAULT-CODE')
```

Comment:
This code should be unique. The uniqueness mechanism is developing in scope of [MODFISTO-83](https://issues.folio.org/browse/MODFISTO-83).